.github/workflows/standards.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - name: Set up Node v14
         uses: actions/setup-node@v2
         with:
-          node-version: '14'
+          node-version: "14"
```

Comment:
If you're running this job with a `node` dependency, does that mean you have access to a JS runtime in this job, and also in `upstream.sh`? If so, you could rely on JS  to manipulate the json response and validate it (see comments below) and remove the python dep if you prefer. Up to you.